### DISCUSSION TITLE
How to play RTSP streaming using MediaElement?

### DISCUSSION THREAD

**Discussion by gandhimonil** [CommunityToolkit/Maui#1131] [Q&A]
Currently, I am working on a streaming project and we have to support RTSP and HLS streaming. I am able to stream both in the Android application but as iOS is using AVPlayer and AVPlayer is not supporting RTSP streaming, I am not able to play RTSP streaming with MediaElement. So please suggest any way to support RTSP streaming for iOS.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
